src/main/java/wooteco/subway/admin/controller/IndexController.java

```diff
@@ -0,0 +1,27 @@
+package wooteco.subway.admin.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@Controller
+public class IndexController {
+    @GetMapping("/")
+    public String index() {
+        return "index";
+    }
+
+    @GetMapping("/adminLine")
```

Comment:
```suggestion
    @GetMapping("/admin-line")
```
url은 소문자를 사용하고 -로 구분하는 걸 권장합니다